completion/zsh/_tlp-run-on

```diff
@@ -0,0 +1,4 @@
+#compdef run-on-ac run-on-bat
+
+# Command line is an arbitrary command, so just forward completion to _normal
+_normal
```

Comment:
When I press tab twice after entering `run-on-ac` or `run-on-bat` I get 
  
> _normal:4: maximum nested function level reached; increase FUNCNEST?
_normal:4: maximum nested function level reached; increase FUNCNEST?
_normal:4: maximum nested function level reached; increase FUNCNEST?
_normal:4: maximum nested function level reached; increase FUNCNEST?

Reply:
huh, it happens for me when I try it in a clean docker container with zsh in it, but not in my fully "kitted out" standard setup. Some further investigation will be required!